### PR TITLE
Add Alembic migrations

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by stefanv** [comment]
This is ready for review.

File: doc/migrations.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Setting up
+
+If you are planning to use database migrations, you need to let
+Alembic know the current state of the database.
+
+Presuming you've just started off by running `make load_demo_data`,
+tell Alembic that you are on the latest database schema:
+
+```
+PYTHONPATH=. alembic stamp head
```
Comment (dannygoldstein) [inline doc/migrations.md:12]: Why is `PYTHONPATH=.` needed if this command is being run from `skyportal/`?

**Review by dannygoldstein — APPROVED** [review]

**Review by profjsb — APPROVED** [review]
Looks good. In the documentation, should be clear when the alembic command is run relative to the git commit workflow. Is it done after a commit or before (e.g.)?

**Comment by dannygoldstein** [comment]
merging this as it is in good enough shape, has 2 green checks, was discussed at the stand up this morning, and is needed to ease future redeployments on fritz

**Comment by stefanv** [comment]
Woops, sorry, I'm only getting back to this now.  I will address the feedback and open another PR.

File: Makefile [hunk 2]
```diff
@@ -34,7 +34,7 @@ docker-local: ## Build docker images locally
 doc_reqs:
 	pip install -q -r requirements.docs.txt
 
-api-docs: FLAGS := $(if $(FLAGS),$(FLAGS),"--config=config.yaml")
+api-docs: FLAGS := $(if $(FLAGS),$(FLAGS),--config=config.yaml)
```
[the remark this answers is not in the document]
Comment (stefanv) [reply Makefile:37]: Fortunately, this is a different issue, should be okay here.

File: alembic.ini [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+# timezone to use when rendering the date
+# within the migration file as well as the filename.
+# string value is passed to dateutil.tz.gettz()
+# leave blank for localtime
+# timezone =
+
+# max length of characters to apply to the
+# "slug" field
+# truncate_slug_length = 40
```
[the remark this answers is not in the document]
Comment (stefanv) [reply alembic.ini:18]: Fixed.

File: doc/migrations.md (hunk 1 above)
Comment (stefanv) [reply doc/migrations.md:12]: Not sure; doesn't work without it, though.

**Pull request merged**